profile/Biobank.StructureDefinition.json

```diff
@@ -0,0 +1,262 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "http://bbmri.de/fhir/StructureDefinition/Biobank",
+  "name": "Biobank",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Organization",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Organization",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Organization.extension",
+        "path": "Organization.extension",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "url"
+            }
+          ],
+          "rules": "open"
+        }
+      },
+      {
+        "id": "Organization.extension:description",
+        "path": "Organization.extension",
+        "sliceName": "description",
+        "definition": "Description of the biobank in english.",
+        "min": 1,
+        "max": "1",
+        "type": [
+          {
+            "code": "Extension",
+            "profile": [
+              "http://bbmri-eric.de/fhir/StructureDefinition/organizationDescription"
+            ]
+          }
+        ],
+        "mustSupport": true
+      },
+      {
+        "id": "Organization.extension:juridicalPerson",
+        "path": "Organization.extension",
+        "sliceName": "juridicalPerson",
+        "min": 1,
+        "max": "1",
+        "type": [
+          {
+            "code": "Extension",
+            "profile": [
+              "http://bbmri.de/fhir/StructureDefinition/JuridicalPersion"
+            ]
+          }
+        ],
+        "mustSupport": true
+      },
+      {
+        "id": "Organization.identifier",
+        "path": "Organization.identifier",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "system"
+            }
+          ],
+          "rules": "open"
+        },
+        "min": 1
+      },
+      {
+        "id": "Organization.identifier:Bbmri-EricId",
+        "path": "Organization.identifier",
+        "sliceName": "Bbmri-EricId",
+        "min": 1,
+        "max": "1",
+        "mustSupport": true
+      },
```

Comment:
Can we enforce the regex with constraints here?

Reply:
We should first clarify whether we want the value to be the full identifier including the bbmri-eric:ID... prefix or just the local id. What's your opinion on this?

Reply:
I had a look whether an Identifier can be a URI. But it can't. So we wouldn't benefit from having the full URI here. So yes I would use the local part only here.

Reply:
Ok, i added a contraint for the local part.